NotificationService/AppGroupUserDefaultsStore.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AppGroupUserDefaultsStore.swift
+//  RealifeTech-SDK
+//
+//  Created by YOU-HSUAN YU on 2022/6/27.
+//
+
+import Foundation
+
+class AppGroupUserDefaultsStore {
+
+    private let appGroupStore: CodableStore
+    private let configurationKey = "SDKConfiguration"
+
+    init(appGroupId: String) {
+        let storage = UserDefaultsStorage(userDefaults: UserDefaults(suiteName: appGroupId) ?? .standard)
```

Comment:
Is there any reason to `.standard` as fallback? Maybe this should be a failable initialiser?

Reply:
That's a good point, thank you!